paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/resources/base/BasicResourceItem.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.internal.resources.base
+
+import com.android.ide.common.rendering.api.ResourceValue
+import com.android.ide.common.resources.ResourceItemWithVisibility
+
+/**
+ * Ported from: [BasicResourceItem.java](https://cs.android.com/android-studio/platform/tools/base/+/47d204001bf0cb6273d8b135c7eece3a982cf0e0:resource-repository/main/java/com/android/resources/base/BasicResourceItem.java)
+ *
+ * Combination of [ResourceItemWithVisibility] and [ResourceValue] interfaces.
+ */
+interface BasicResourceItem : ResourceItemWithVisibility, ResourceValue
```

Comment:
delete?  this is app.cash.paparazzi.internal.resources.BasicResourceItem!

Reply:
@jrodbx As discussed, I will keep the existing app.cash.paparazzi.internal.resources.BasicResourceItem for now as it has been integrated with `RepositoryLoader`.  Will delete it and migrate to this new one when overall porting is done